pyccl/nl_pt/power_deprecated.py

```diff
@@ -62,6 +64,10 @@ def __init__(self, with_NC=False, with_IA=False, with_dd=True,
                  pad_factor=1, low_extrap=-5, high_extrap=3,
                  P_window=None, C_window=.75,
                  k_cutoff=None, n_exp_cutoff=4):
+        warnings.warn("PTCalculators are deprecated and will be removed in "
+                      "future versions. Use the new perturbation theory "
+                      "calculators (e.g. EulerianPTCalculator).",
+                      CCLDeprecationWarning)
```

Comment:
Not a good idea to rename the module like this and fill it with all those warnings - I think it's best to just wrap the `__init__` function and all other functions that will be deprecated with the `@deprecated` decorator. Otherwise it just breaks API.

Reply:
How does it break the API? The exact same functionality is still there. This will just throw warnings whenever users try to call the deprecated function or initialize the deprecated class.

Reply:
The module is renamed. To preserve the API you're going to have to add it to the renaming dictionary in `__getattr__` in `__init__.py`, which complicates things. Easiest way is to add `@deprecated` to `__init__()`.

Reply:
yep that does it